Documentation/Signoffs/Microcontroller.md

```diff
@@ -0,0 +1,55 @@
+9. Some form of battery monitoring capabilities need to be included or applicable.  Telling the user how much longer the device may operate is a must.  This can be done using an additional component or algorithmic measurement.  The device will need to keep track of battery life to prevent data loss by saving and ceasing operations when reaching critical levels.  Doing this will ensure data protection.  
+10. The board itself cannot be too big.  “Too big” here is defined as having a volume greater than 50 cubic inches.  This size was set to prevent the device from proving too large.  50 cubic inches would be about the size of a small novella/book.  .  
+11. The manufacturer needs to be a reputable and safe choice.  Choosing a bad manufacturer could hurt the project.  A stable choice will lower the risk of receiving a malfunctioning board or running out of the specified board supply.  This device is planned to be open to public use and construction, so it will need to be trustworthy.
+12. The board will need to have coding language support for assembly, C, and or Python.  It is expected that the users will have some degree of coding experience and may potentially personalize or improve upon the code. These languages are chosen due to popularity and performance.  C, in particular, is predicted to be very desirable for this project.
+
+## BUILDABLE SCHEMATIC OF SUBSYSTEM
+Shown is the pseudo-code for the operating code.  The final form of the code may have parts added and additional functionalities included, but this is the general idea of how the code should flow.  Proceeding from each section is assuming that there are no errors detected indicating a “Success”.  If an error arises, a default handler will need to be called to assess the situation.  It should also be noted that the loop is intended to be theoretically infinite, however, there are explicit and implicit breaks at the battery running critically low (explicit) and the shutoff button being pushed (implicit).  “Timers” refer to either physical hardware or coded timers that go off to alert the system that the sensor(s) sampling is ready.
+
+![pseudo code 2](https://user-images.githubusercontent.com/110966922/202583040-c28e27fe-c175-4287-a8cb-cefafbfe327b.png)
+
+## ANALYSIS OF CHOSEN COMPONENT(S)
+The team has chosen to look at the Arduino Mega 2560 Rev 3 and Arduino USB Shield Host as the solutions.  The Arduino Mega is a larger and more powerful version of the Arduino Uno with much more storage, memory, and I/O pins.  The USB Shield is a breakout board or additional component that connects to the Arduino Mega to allow for USB devices to be implemented and utilized.  These components, in conjunction, will satisfy the constraints and jobs of this subsystem.  Following are the corresponding fulfillment of the constraints:
```

Comment:
This section shows that the chosen devices satisfies the constraints. Each item in this list should correspond to an item in the constraints list. Again, eliminate unnecessary prose, for example see comment in #4. I will not comment on all these at this time....